src/lib/components/Card/Card.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  import "./Card.scss";
+  import Button from "$lib/components/Button";
+  import Image from "$lib/components/Image";
+  import classNames from "$lib/util/classNames";
+  import type { Variant } from "./cardOptions";
+
+  export let variant: Variant = "primary";
+
+  const variantClassesDict: Record<Variant, string> = {
+    primary: $$slots.image ? "usa-card--flag" : "",
+    secondary: "",
+  };
+  $: variantClass = variantClassesDict[variant];
+
+  let className = "";
+  export { className as class };
+  $: classes = classNames("usa-card", variantClass, className);
+</script>
+
+<li class={`${classes}`}>
```

Comment:
nit: this can just be simplified to this right?
```suggestion
<li class={classes}>
```

Reply:
it sure can 🤦